testdata/deployment/OCP-10873/dc_new.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: nginx-config
+  data:
+    nginx.conf: |
+      events {
+          worker_connections 1024;
+      }
+
+      http {
+          server {
+              listen		8080;
+              location / {
+                  root /data/http;
+              }
+          }
+
+          server {
+              listen           	    8443 ssl http2 default;
+              listen           	    [::]:8443 ssl http2 default;              
+              server_name      	    _;
+              ssl_certificate  	    certs/tls.crt;
+              ssl_certificate_key  	certs/tls.key;
+              location / {
+                  root /data/https-default;
+              }
+          }
+      }
+- apiVersion: v1
+  kind: ReplicationController
+  metadata:
+    labels:
+      name: web-server-rc
+    name: web-server-rc
+  spec:
+    replicas: 1
+    template:
+      metadata:
+        labels:
+          name: web-server-rc
+      spec:
+        containers:
+        - name: nginx
+          image: quay.io/openshifttest/nginx-alpine@sha256:f266733786efb10c4353d4b44ada6f22434e983d22f0975d20803f1817d38f56
```

Comment:
It is better to use tag `quay.io/openshifttest/nginx-alpine:multiarch` , we also need to support arm platform.

Reply:
Seems this is not good. "sha256" images are widely used in tests. Without this, in disconnected envs, we can't test such cases, because images will hit ImagePullBackOff. But with sha256, which is called image digest, pod can be Running in test runs of disconnected envs. See the doc https://url.corp.redhat.com/b402018 written by Xiujuan/Wenjing

Reply:
As long as the sha256 digest is the digest value of the multiarch image, your concern would be not an issue.

Reply:
@xxia Thanks for the information.